bin/process_id.json

```diff
@@ -0,0 +1,5 @@
+{
+    "docservice": -1,
+    "spellchecker": -1,
+    "converter" : -1
```

Comment:
@GaneshAnarse I think we need to remove spellchecker

Reply:
Yup.
Removed the default spellchecker PID.